src/client/hud.cpp

```diff
@@ -415,8 +447,13 @@ void Hud::drawStatbar(v2s32 pos, u16 corner, u16 drawdir, const std::string &tex
 	core::dimension2di dstd;
 	if (size == v2s32()) {
 		dstd = srcd;
+#ifdef __ANDROID__ // Android and Desktop
+		dstd.Height *= size_factor;
+		dstd.Width  *= size_factor;
+		offset.X *= size_factor;
+		offset.Y *= size_factor;
+#endif
 	} else {
-		float size_factor = m_hud_scaling * RenderingEngine::getDisplayDensity();
```

Comment:
Please keep as it was before.

Reply:
What for? Call porting again?
We get this value at the beginning.

Reply:
Forgot to remove this comment after thinking about possible solutions to make this code easier to read.
Stale comment.